docs/ref_functions.md

```diff
@@ -117,3 +117,71 @@ if (timer_elapsed(key_timer) < 100) {
   // do something if 100ms or more have passed
 }
 ```
+## Custom modified keycode handling
+
+Sometimes you might want to send a specific keycode or execute a specific macro
+when some modifiers and a key are pressed. For example let's say that your
+keymap binds a shifted keycode like question mark (?) on a qwerty keyboard to a
+key. When pressed, qmk will send these events: shift pressed, slash (/) pressed,
+slash released, shift released. But then what happens when pressing shift and
+that key ? Qmk will send shitf pressed, slash pressed, slash released and shift
+released. The OS won't be able to distinguish between the two and you might just
+have lost a usefull keycombo on your shiny new keymap. This is  where the
+`custom_keycode_on_modifiers` function comes in. It can be used to have qmk send
+another keycode when some modifiers are pressed. Its prototype is
+
+```c
+bool custom_keycode_on_modifiers(uint8_t modifier_mask, uint8_t layer, keyrecord_t *record, uint16_t custom_keycode)
+```
+
+The first argument is the modifiers that must be held for the custom keycode to
```

Comment:
I think this may be better if you present a use case for this, first, and then how to do it in the code. 

Also, again, could you make this block one line?

Reply:
I'm not following you here. The uses case is the first thing I explain in the new documentation on line 120. Is that not enough ? Or am I missing something ?

Reply:
ping